src/SearchbarPageItem.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div v-if="item.heading" class="heading">
+    <div class="heading-text">{{ item.heading.text }}</div>
+    <div class="heading-text-items">
+      <small v-html="highlight(item.heading.text, value)"></small>
+      <br />
+      <small v-for="(keyword, index) in item.keywords" :key="index">
+        <span v-html="highlight(keyword, value)"></span>
+        <br />
+      </small>
+    </div>
+  </div>
+  <div v-else>
+    <span class="page-title" v-html="highlight(item.title, value)"></span>
+    <br v-if="item.keywords" />
+    <small v-if="item.keywords" v-html="highlight(item.keywords, value)"></small>
+    <hr class="page-headings-separator"/>
```

Comment:
@ang-zeyu oops, I missed this just now. There seems to be an inconsistent spacing here between the class and the closing `/>`